itools-packager/src/main/java/com/powsybl/itools/ItoolsPackagerMojo.java

```diff
@@ -100,6 +107,37 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
         }
     }
 
+    private void targz(Path dir, Path baseDir, Path zipFilePath) throws IOException {
+        getLog().info("Targz package");
+        try (TarArchiveOutputStream zos =
+                new TarArchiveOutputStream(new GzipCompressorOutputStream(Files.newOutputStream(zipFilePath)))) {
+            zos.setLongFileMode(TarArchiveOutputStream.LONGFILE_POSIX); // allow long file paths
```

Comment:
The test is failing on my computer with the following error:
```
java.lang.IllegalArgumentException: group id '17.......4' is too big ( > 2097151 ). Use STAR or POSIX extensions to overcome this limit
```
<br/>

I have to add the following option to successfully run the test. Is it OK for you to add it?

```java
zos.setBigNumberMode(TarArchiveOutputStream.BIGNUMBER_POSIX); // allow large numbers (for instance a big GID)
```

Reply:
Nicely spotted, I'll add it yes 👍